src/objects/Application__c.object

```diff
@@ -0,0 +1,268 @@
+    <enableSharing>true</enableSharing>
+    <enableStreamingApi>true</enableStreamingApi>
+    <fields>
+        <fullName>Applicant_Decision_Date__c</fullName>
+        <description>The date when the student decides whether to enroll.</description>
+        <externalId>false</externalId>
+        <inlineHelpText>The date when the student decides whether to enroll.</inlineHelpText>
+        <label>Applicant Decision Date</label>
+        <required>false</required>
+        <trackHistory>false</trackHistory>
+        <trackTrending>false</trackTrending>
+        <type>Date</type>
+    </fields>
+    <fields>
+        <fullName>Applicant__c</fullName>
+        <deleteConstraint>SetNull</deleteConstraint>
+        <description>The Contact applying to the institution or program.</description>
+        <externalId>false</externalId>
+        <inlineHelpText>The Contact applying to the institution or program.</inlineHelpText>
+        <label>Applicant</label>
+        <referenceTo>Contact</referenceTo>
+        <relationshipLabel>Applications</relationshipLabel>
+        <relationshipName>Applications</relationshipName>
+        <required>false</required>
+        <trackHistory>false</trackHistory>
+        <trackTrending>false</trackTrending>
+        <type>Lookup</type>
+    </fields>
+    <fields>
+        <fullName>Application_Date__c</fullName>
+        <description>The date and time when the application was submitted.</description>
+        <externalId>false</externalId>
+        <inlineHelpText>The date and time when the application was submitted.</inlineHelpText>
+        <label>Application Date</label>
+        <required>false</required>
+        <trackHistory>false</trackHistory>
+        <trackTrending>false</trackTrending>
+        <type>DateTime</type>
+    </fields>
+    <fields>
+        <fullName>Application_Decision_Date__c</fullName>
+        <description>The date when the institution decides whether to accept the applicant.</description>
+        <externalId>false</externalId>
+        <inlineHelpText>The date when the institution decides whether to accept the applicant.</inlineHelpText>
+        <label>Application Decision Date</label>
+        <required>false</required>
+        <trackHistory>false</trackHistory>
+        <trackTrending>false</trackTrending>
+        <type>Date</type>
+    </fields>
+    <fields>
+        <fullName>Application_Status__c</fullName>
+        <description>The status of the application review process.</description>
+        <externalId>false</externalId>
+        <inlineHelpText>The status of the application review process.</inlineHelpText>
+        <label>Application Status</label>
+        <required>false</required>
+        <trackHistory>false</trackHistory>
+        <trackTrending>false</trackTrending>
+        <type>Picklist</type>
+        <valueSet>
+            <restricted>true</restricted>
+            <valueSetDefinition>
+                <sorted>false</sorted>
```

Comment:
Comments in the spec suggest this should be sorted alphabetically.

Reply:
Oh I missed that comment, thanks for catching it.